pep-0001.txt

```diff
@@ -229,45 +255,69 @@ PEP Review & Resolution
+style and consistency from the PEP editors.
+
+However, content review and final acceptance of the PEP must be requested of the
+core developers, usually via an email to the python-dev mailing list.
+
+To expedite the process in selected cases (e.g. when a change is clearly
+beneficial and ready to be accepted, but the PEP hasn't been formally submitted
+for review yet), the Steering Council may also initiate a PEP review, first
+notifying the PEP author(s) and giving them a chance to make revisions.
+
+The final authority for PEP approval is the Steering Council. However, whenever
+a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the BDFL's delegate (or "PEP czar") for that PEP. If their self-nomination
-is accepted by the other core developers and the BDFL, then they will have
-the authority to approve (or reject) that PEP. This process happens most
-frequently with PEPs where the BDFL has granted in principle approval for
-*something* to be done, but there are details that need to be worked out
-before the PEP can be accepted.
-
-If the final decision on a PEP is to be made by a delegate rather than
-directly by the BDFL, this will be recorded by including the
-"BDFL-Delegate" header in the PEP.
-
-PEP review and resolution may also occur on a list other than python-dev
-(for example, distutils-sig for packaging related PEPs that don't
-immediately affect the standard library). In this case, the "Discussions-To"
-heading in the PEP will identify the appropriate alternative list where
-discussion, review and pronouncement on the PEP will occur.
+the BDFL-Delegate for that PEP, and they will then have the authority to approve
+(or reject) that PEP.
+
+The designated decision maker for each PEP is recorded in the "BDFL-Delegate"
+header in the PEP.
+
+Such self-nominations are accepted by default, but may be explicitly declined by
+the Steering Council. Possible reasons for the Steering Council declining a
+self-nomination as BDFL-Delegate include, but are not limited to, perceptions of
+a potential conflict of interest (e.g. working for the same organisation as the
+PEP submitter), or simply considering another potential BDFL-Delegate to be
+more appropriate. If core developers (or other community members) have concerns
+regarding the suitability of a BDFL-Delegate for any given PEP, they may ask
+the Steering Council to review the delegation.
+
+If no volunteer steps forward, then the Steering Council will approach core
+developers (and potentially other Python community members) with relevant
+expertise, in an attempt to identify a candidate that is willing to serve as
+BDFL-Delegate for that PEP. If no suitable candidate can be found, then the
+PEP will be marked as Deferred until one is available.
+
+While BDFL-Delegates hold final authority over acceptance or rejection of a PEP,
```

Comment:
I wonder if we should ratification by the SC in the PEP approval process. I could just see a rogue BDFL-Delegate announcing approval of a PEP before they are asked to step down by the SC (or perhaps even refusing to step down). It would be nice if the PEP made clear that in this case the SC wins. (If the SC itself has gone rogue, PEP 13 has ways to deal with that.)

Reply:
+1 - I don't think it's too much of a burden for the SC to ratify.  It'll usually be rubber stamped but I think it does give a little extra legitimacy and checks-n-balances on the process.

Reply:
+1

Reply:
I agree the handling of this case needs to be clearer.

In https://github.com/python/peps/pull/896/commits/bad24f3c57dfb9f97a62d931c2f058b3db3166c0, rather than adding a ratification step, I instead made it so that if a BDFL-Delegate is asked to stand down by the Council, any prior decision about acceptance or rejection is reverted (and that reversion is triggered by the request itself, as if the SC were happy with the decision that had already been made, they wouldn't be asking the delegate to stand down).

If folks have a strong preference for routine ratification, then I can find a way to reword that section. However, I do think going down that path would mostly just be annoying for BDFL-Delegates and Steering Council members - in a work context, I've been in the position of both seeking the rubberstamp and of wielding it, and it irks me no matter which side I'm on.